stable/mission-control/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: mission-control
 description: A Helm chart for JFrog Mission Control
-version: 2.0.1
+version: 2.0.2
```

Comment:
@rahulsadanandan can you please add this chart version in chart install step in readme file

Reply:
Its done now